charts/celery-exporter/Chart.yaml

```diff
@@ -12,6 +12,4 @@ sources:
 maintainers:
   - name: danihodovic
   - name: adinhodovic
-
 version: 0.7.0
-appVersion: 0.9.2
```

Comment:
Preferring to set the app-version dynamically i.e. with the `--app-version` flag available on `helm package` ...

```console
$ helm package --help | grep '\-version'
      --app-version string       set the appVersion on the chart to this version
      --version string           set the version on the chart to this semver version
```